connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/SparkSessionSuite.scala

```diff
@@ -73,4 +78,21 @@ class SparkSessionSuite extends ConnectFunSuite {
       session2.close()
     }
   }
+
+  class CustomChannelBuilder() extends ChannelBuilder() {
+    override def createChannelBuilder(
+        host: String,
+        port: Int,
+        credentials: ChannelCredentials): ManagedChannelBuilder[_] = {
+
+      val builder = super.createChannelBuilder(host, port, credentials)
+      builder.userAgent("Custom agent")
+      builder
+    }
+  }
+
+  test("channelBuilder") {
+    val channelBuilder = new CustomChannelBuilder()
+    SparkSession.builder().remote("sc://test.it:16845").channelBuilder(channelBuilder).create()
```

Comment:
missing assertion.

Reply:
What would you like to assert here?

Reply:
I assume you've created CustomChannelBuilder and setting user agent with the intent to verify that this actually shows up somewhere, no?